collections/_sdk/sandboxing-and-allowed-imports.md

```diff
@@ -0,0 +1,212 @@
+---
+title: "Sandboxing and Allowed Imports"
+---
+
+Plugins developed with the Canvas SDK execute safely and securely in a sandbox that restricts access to the host operating system, filesystem, and database. This precaution reduces the likelihood of accidents or malicious use of the platform that might put patient data at risk.
+
+## Standard Library Modules
+
+The following Python standard library modules and their allowed imports are available within the sandbox:
+
+### `__future__`
```

Comment:
This is too big on the page. Replace all uses of `###` on this page with `#####` instead.

For each Python standard library module, add a sentence or two about what it is used for, and link to the official docs for that library.